[00:18:18.892] Yuebon.AspNetCore.Mvc.Filter.ExceptionHandlingAttribute T8 系统警告信息
System.ArgumentOutOfRangeException: startIndex cannot be larger than length of string. (Parameter 'startIndex')
   at System.String.Substring(Int32 startIndex, Int32 length)
   at System.String.Substring(Int32 startIndex)
   at Yuebon.AspNetCore.Controllers.ApiController.CheckToken() in D:\netcorespace\YueGroup\src\Yuebon.AspNetCore\Mvc\ApiController.cs:line 62
   at Yuebon.WebApi.Areas.WeiXin.Controllers.WxOpenController.QuikLogin(WxUserInfo info) in D:\netcorespace\YueGroup\src\Yuebon.WebApi\Areas\WeiXin\Controllers\WxOpenController.cs:line 405
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[00:21:31.174] Yuebon.AspNetCore.Mvc.Filter.ExceptionHandlingAttribute T19 系统警告信息
System.NullReferenceException: Object reference not set to an instance of an object.
   at Yuebon.Commons.Helpers.HttpContextHelper.get_HttpContext() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\HttpContextHelper.cs:line 19
   at Yuebon.Commons.Helpers.SessionHelper.get_HttpHelper() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 15
   at Yuebon.Commons.Helpers.SessionHelper.GetString(String key) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 44
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.OperationLogOfUpdate(T obj, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 1063
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.Update(T entity, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 684
   at Yuebon.Commons.Core.Services.BaseService`2.Update(T entity, Tkey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Services\BaseService.cs:line 273
   at Yuebon.Security.Services.UserService.Yuebon.Commons.Core.IServices.IService<Yuebon.Security.Models.User,System.String>.Update(User entity, String id, IDbTransaction trans)
   at Yuebon.Security.Application.UserApp.UpdateUserByOpenId(UserInPutDto userInPut) in D:\netcorespace\YueGroup\src\Yuebon.Security.Core\Application\UserApp.cs:line 129
   at Yuebon.WebApi.Areas.WeiXin.Controllers.WxOpenController.QuikLogin(WxUserInfo info) in D:\netcorespace\YueGroup\src\Yuebon.WebApi\Areas\WeiXin\Controllers\WxOpenController.cs:line 428
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[00:22:06.068] Yuebon.AspNetCore.Mvc.Filter.ExceptionHandlingAttribute T19 系统警告信息
System.NullReferenceException: Object reference not set to an instance of an object.
   at Yuebon.Commons.Helpers.HttpContextHelper.get_HttpContext() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\HttpContextHelper.cs:line 19
   at Yuebon.Commons.Helpers.SessionHelper.get_HttpHelper() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 15
   at Yuebon.Commons.Helpers.SessionHelper.GetString(String key) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 44
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.OperationLogOfUpdate(T obj, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 1063
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.Update(T entity, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 684
   at Yuebon.Commons.Core.Services.BaseService`2.Update(T entity, Tkey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Services\BaseService.cs:line 273
   at Yuebon.Security.Services.UserService.Yuebon.Commons.Core.IServices.IService<Yuebon.Security.Models.User,System.String>.Update(User entity, String id, IDbTransaction trans)
   at Yuebon.Security.Application.UserApp.UpdateUserByOpenId(UserInPutDto userInPut) in D:\netcorespace\YueGroup\src\Yuebon.Security.Core\Application\UserApp.cs:line 129
   at Yuebon.WebApi.Areas.WeiXin.Controllers.WxOpenController.QuikLogin(WxUserInfo info) in D:\netcorespace\YueGroup\src\Yuebon.WebApi\Areas\WeiXin\Controllers\WxOpenController.cs:line 428
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[00:33:42.430] Yuebon.AspNetCore.Mvc.Filter.ExceptionHandlingAttribute T4 系统警告信息
System.NullReferenceException: Object reference not set to an instance of an object.
   at Yuebon.Commons.Helpers.HttpContextHelper.get_HttpContext() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\HttpContextHelper.cs:line 19
   at Yuebon.Commons.Helpers.SessionHelper.get_HttpHelper() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 15
   at Yuebon.Commons.Helpers.SessionHelper.GetString(String key) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 44
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.OperationLogOfUpdate(T obj, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 1063
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.Update(T entity, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 684
   at Yuebon.Commons.Core.Services.BaseService`2.Update(T entity, Tkey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Services\BaseService.cs:line 273
   at Yuebon.Security.Services.UserService.Yuebon.Commons.Core.IServices.IService<Yuebon.Security.Models.User,System.String>.Update(User entity, String id, IDbTransaction trans)
   at Yuebon.Security.Application.UserApp.UpdateUserByOpenId(UserInPutDto userInPut) in D:\netcorespace\YueGroup\src\Yuebon.Security.Core\Application\UserApp.cs:line 129
   at Yuebon.WebApi.Areas.WeiXin.Controllers.WxOpenController.QuikLogin(WxUserInfo info) in D:\netcorespace\YueGroup\src\Yuebon.WebApi\Areas\WeiXin\Controllers\WxOpenController.cs:line 428
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[00:34:39.520] Yuebon.AspNetCore.Mvc.Filter.ExceptionHandlingAttribute T46 系统警告信息
System.NullReferenceException: Object reference not set to an instance of an object.
   at Yuebon.Commons.Helpers.HttpContextHelper.get_HttpContext() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\HttpContextHelper.cs:line 19
   at Yuebon.Commons.Helpers.SessionHelper.get_HttpHelper() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 15
   at Yuebon.Commons.Helpers.SessionHelper.GetString(String key) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 44
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.OperationLogOfUpdate(T obj, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 1063
   at Yuebon.Commons.Core.Repositories.BaseRepository`2.Update(T entity, TKey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Repositories\BaseRepository.cs:line 684
   at Yuebon.Commons.Core.Services.BaseService`2.Update(T entity, Tkey id, IDbTransaction trans) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Core\Services\BaseService.cs:line 273
   at Yuebon.Security.Services.UserService.Yuebon.Commons.Core.IServices.IService<Yuebon.Security.Models.User,System.String>.Update(User entity, String id, IDbTransaction trans)
   at Yuebon.Security.Application.UserApp.UpdateUserByOpenId(UserInPutDto userInPut) in D:\netcorespace\YueGroup\src\Yuebon.Security.Core\Application\UserApp.cs:line 129
   at Yuebon.WebApi.Areas.WeiXin.Controllers.WxOpenController.QuikLogin(WxUserInfo info) in D:\netcorespace\YueGroup\src\Yuebon.WebApi\Areas\WeiXin\Controllers\WxOpenController.cs:line 428
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[00:46:34.325] Yuebon.AspNetCore.Mvc.Filter.ExceptionHandlingAttribute T5 系统警告信息
System.NullReferenceException: Object reference not set to an instance of an object.
   at Yuebon.Commons.Helpers.HttpContextHelper.get_HttpContext() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\HttpContextHelper.cs:line 19
   at Yuebon.Commons.Helpers.SessionHelper.get_HttpHelper() in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 15
   at Yuebon.Commons.Helpers.SessionHelper.SetSession(String key, Object value) in D:\netcorespace\YueGroup\src\Yuebon.Commons\Helpers\SessionHelper.cs:line 23
   at Yuebon.WebApi.Areas.WeiXin.Controllers.WxOpenController.QuikLogin(WxUserInfo info) in D:\netcorespace\YueGroup\src\Yuebon.WebApi\Areas\WeiXin\Controllers\WxOpenController.cs:line 448
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
[01:04:18.446] Yuebon.AspNetCore.Mvc.Filter.ExceptionHandlingAttribute T77 系统警告信息
System.ArgumentOutOfRangeException: startIndex cannot be larger than length of string. (Parameter 'startIndex')
   at System.String.Substring(Int32 startIndex, Int32 length)
   at Yuebon.AspNetCore.Controllers.ApiController.CheckToken() in D:\netcorespace\YueGroup\src\Yuebon.AspNetCore\Mvc\ApiController.cs:line 62
   at Yuebon.WebApi.Areas.WeiXin.Controllers.WxOpenController.GetOpenId(String code) in D:\netcorespace\YueGroup\src\Yuebon.WebApi\Areas\WeiXin\Controllers\WxOpenController.cs:line 167
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
